x/cork/types/proposal.go

```diff
@@ -7,13 +7,22 @@ import (
 )
 
 const (
-	ProposalTypeAddManagedCellars    = "AddManagedCellars"
-	ProposalTypeRemoveManagedCellars = "RemoveManagedCellars"
+	ProposalTypeAddManagedCellars    = "AddManagedCellarIDs"
+	ProposalTypeRemoveManagedCellars = "RemoveManagedCellarIDs"
 )
 
 var _ govtypes.Content = &AddManagedCellarsProposal{}
 var _ govtypes.Content = &RemoveManagedCellarsProposal{}
 
+func init() {
+	govtypes.RegisterProposalType(ProposalTypeAddManagedCellars)
+	govtypes.RegisterProposalTypeCodec(&AddManagedCellarsProposal{}, "sommelier/AddManagedCellarIDsProposal")
```

Comment:
Did the renaming of this fix anything, or was it a red herring? Should we rename the proposal types themselves to match the proposal type string?

Reply:
Yes, it was a real fix. I will change the variables to match